nbconvert/templates/latex/base.tplx

```diff
@@ -46,7 +44,6 @@ This template does not define a docclass, the inheriting class must define this.
     \usepackage{upquote} % Upright quotes for verbatim code
     \usepackage{eurosym} % defines \euro
     \usepackage[mathletters]{ucs} % Extended unicode (utf-8) support
-    \usepackage[utf8x]{inputenc} % Allow utf-8 characters in the tex document
```

Comment:
Good catch. Not only is this ignored on XeLaTeX, the default for PDFLaTeX is now ```\usepackage[utf8]{inputenc}``` (since April 2018) which has been recommended over ```utf8x``` for a while now anyway.